adhocracy4/modules/models.py

```diff
@@ -25,6 +31,42 @@ def settings_instance(self):
                 return getattr(self, setting)
 
     @property
+    def is_active(self):
+        return self.phase_set \
+            .active_phases()\
+            .exists()
+
+    @property
+    def is_active_in_project(self):
+        return self == self.project.active_module
+
+    @property
+    def active_phase(self):
+        return self.phase_set \
+            .active_phases() \
+            .first()
+
+    @functional.cached_property
```

Comment:
not sure if that makes sense, as `all()` returns a `QuerySet` which will be lazy evalauated on the db when it is iterated

```
type(Module.objects.first().phase_set.all())
Out[16]: adhocracy4.phases.models.PhasesQuerySet
```